 BinaryConverter/src/removeDuplicate/removeDuplicateRunner.java

```diff
@@ -0,0 +1,39 @@
+package removeDuplicate;
+import java.util.ArrayList;
+
+public class removeDuplicateRunner {
+	
+	public static ArrayList<String> l = new ArrayList<String>();
+	
+	public static void main(String[] args) {
+		
+		l.add("Carl");
+		l.add("Chris");
+		l.add("Chris");
+		l.add("Cathy");
+		l.add("Carl");
+		l.add("Julian");
+		l.add("Kartik");
+		l.add("Ryan");
+		l.add("Kartik");
+		l.add("Sarah");
+		
+		removeDuplicates(l);
+		
+		System.out.println(l);
+		
+		
+	}
+	
+	public static void removeDuplicates(ArrayList<String> list) {
+		for(int i = 0;i < list.size(); i++) {
+			for(int j = 0; j < list.size(); j++) {
+				if(i!=j && list.get(i) == list.get(j)) {
+					list.remove(j);
+				}
+			}
+			
+		}
+	}
```

Comment:
This works perfectly fine but would be slow. If you know what a big O is, the big O of this function you wrote is O(n^2). There would be a more efficient way to run this that uses an ArrayList to store all the checked ones. The O is O(n) for that function. 
Also, even within this function, there would be a simpler way by just defining `j = i+1` instead of `j=0`
But watch out for the shift of index after you remove something.